app/src/main/java/com/example/poste/models/Folder.java

```diff
@@ -82,12 +81,11 @@ public String getUserPermission() {
     }
 
     /**
-     * Returns a copy of the list of posts in the folder. This is a copy, so modifying the returned
-     * list will not modify the folder's posts.
-     * @return a copy of the list of posts in the folder.
+     * Returns a List of the posts in the folder.
```

Comment:
I like the idea of immutable return values here (returning a copy, instead of the actual list) -- however, I think this was causing some confusion and we were attempting to modify the returned lists of folders / posts. Until we can get on the same page and refactor, this will solve some issues.